### PR TITLE
Net version2

### PR DESCRIPTION
**Pull request by Paolob69**
impl net_version and web3_client_version through ethbackend

### CODE REVIEW

File: silkrpc/commands/web3_api.cpp [hunk 1]
```diff
@@ -29,8 +29,16 @@ namespace silkrpc::commands {
 
 // https://eth.wiki/json-rpc/API#web3_clientversion
 asio::awaitable<void> Web3RpcApi::handle_web3_client_version(const nlohmann::json& request, nlohmann::json& reply) {
-    reply = make_json_content(request["id"], common::kEthereumNodeName);
```
Comment (canepat) [inline silkrpc/commands/web3_api.cpp:32]: Now you can delete the following lines in file `constants.hpp`:
```
constexpr const char* kNodeName{"TurboGeth"};
constexpr const char* kVersion{"2021.05.3-alpha"};
const std::string kEthereumNodeName = boost::str(boost::format("%1%/v%2%/%3%") % kNodeName % kVersion % kOsName); // NOLINT

constexpr uint64_t kETH66{66};
```

File: silkrpc/ethbackend/backend.hpp [hunk 2]
```diff
@@ -83,6 +106,24 @@ class BackEnd final {
...
+        const auto nv = reply.id();
+        SILKRPC_DEBUG << "BackEnd::net_version version=" << nv << " t=" << clock_time::since(start_time) << "\n";
+        co_return nv;
+    }
+
+    asio::awaitable<std::string> get_client_version() {
+        const auto start_time = clock_time::now();
+        const auto reply = co_await cv_awaitable_.async_call(asio::use_awaitable);
+        const auto cv = reply.nodename();
+        SILKRPC_DEBUG << "BackEnd::web3_client_version version=" << cv << " t=" << clock_time::since(start_time) << "\n";
```
Comment (canepat) [inline silkrpc/ethbackend/backend.hpp:121]: `BackEnd::web3_client_version` -> `BackEnd::get_client_version`

File: silkrpc/ethbackend/backend.hpp [hunk 3]
```diff
@@ -83,6 +106,24 @@ class BackEnd final {
         co_return pv;
     }
 
+    asio::awaitable<uint64_t> get_net_version() {
+        const auto start_time = clock_time::now();
+        const auto reply = co_await nv_awaitable_.async_call(asio::use_awaitable);
+        const auto nv = reply.id();
+        SILKRPC_DEBUG << "BackEnd::net_version version=" << nv << " t=" << clock_time::since(start_time) << "\n";
```
Comment (canepat) [inline silkrpc/ethbackend/backend.hpp:113]: `BackEnd::net_version` -> `BackEnd::get_net_version`

**Review by canepat — CHANGES_REQUESTED** [review]

File: silkrpc/ethbackend/backend.hpp (hunk 2 above)
Comment (Paolob69) [reply silkrpc/ethbackend/backend.hpp:121]: ok

File: silkrpc/ethbackend/backend.hpp (hunk 3 above)
Comment (Paolob69) [reply silkrpc/ethbackend/backend.hpp:113]: ok

File: silkrpc/commands/web3_api.cpp (hunk 1 above)
Comment (Paolob69) [reply silkrpc/commands/web3_api.cpp:32]: ok

**Pull request merged**